notebooks/book1/02/change_of_vars_demo1d.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
<p>In this cell, variables are declared badly</p><p>I am giving some examples</p><p>fn -&gt; square</p><p>a -&gt; lower_limit</p><p>b -&gt; upper_limit</p><p>px -&gt; px_uniform</p><p> </p><p>This is process of converting code to PEP8</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
![variables_convention](https://user-images.githubusercontent.com/87682045/169585178-1a16a095-c611-476c-ab83-26c14beacfe8.PNG)